rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -1412,14 +1412,8 @@ error_code sys_net_bnet_poll(ppu_thread& ppu, vm::ptr<sys_net_pollfd> fds, s32 n
 					return {};
 				}
 
-				std::lock_guard nw_lock(g_fxo->get<network_context>().mutex_thread_loop);
-
-				if (signaled)
-				{
-					break;
-				}
-
 				has_timedout = network_clear_queue(ppu);
+				ppu.state -= cpu_flag::signal;
```

Comment:
lmao just continue the loop if it did not timed out.

Reply:
It's there for sys_net_abort

Reply:
I mean it is just a stylistic choice because in the other syscalls, if it needs to remove the signal flag it continues the loop and the would be removed at the beginning of it. But it's not a blocker.